src/tools/bin/tbg

```diff
@@ -357,6 +379,7 @@ cp -a "$TBG_cfgFile" tbg/submit.cfg
 
 #warn, if there are still unresolved !TBG_ variables left
 check_final tbg/submit.start "$tooltpl_file_data"
+transfer_permissions $tplPathAndFile tbg/submit.start
```

Comment:
Just a side note: This does not work because `$tooltpl_file` can be a relative path if it is given as relative path via `-t` in `tbg` and thus will fail at `cp ...`.

Converting  `$tooltpl_file` to an absolute path is thus essential.